pkg/store/pipelineconfig/config_store_test.go

```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package store
+package pipelineconfig
```

Comment:
This is for device "configs" in general, so pipeline, chassis or other firmware potentially. So the name should reflect the more general nature.

Reply:
Good point. I will change it.